### ISSUE TITLE
How to reproduce?

### ISSUE THREAD

**Issue by dru9** [PJLab-ADG/LoGoNet#15]
Thank you for your code!
I tried to train 1-stage model to use this one as pretrained_model in 2-stage. 
So I changed below lines in LoGoNet-5f.yaml

SECOND_STAGE: False
NUM_EPOCHS: 20

But accuracy was quite low. So I'm checking what's the problem.
Am I right to train 1-stage model like this?

**Comment by dru9** [comment]
I missed this words. Thank you for your code!

**Comment by sankin97** [comment]
Sorry for the delayed response. To begin, you will need to train the LiDAR-only detector（5 frames) by running the following command:
`sh scripts/slurm_train.sh ${PARTITION} ${JOB_NAME} ${NUM_GPUS} --cfg_file cfgs/det_model_cfgs/waymo/LoGoNet-1stage-nomm.yaml`

For training 3 frames or 1 frame, you need to modify the `DATA_CONFIG._BASE_CONFIG_`.
After that, you can proceed with training LoGoNet using this command:
`sh scripts/slurm_train.sh ${PARTITION} ${JOB_NAME} ${NUM_GPUS} --cfg_file cfgs/det_model_cfgs/waymo/LoGoNet-5f.yaml --pretrained_model ${PRETRAINED_MODEL_PATH}`

For more guidance on training LoGoNet on the WOD dataset, please refer to the newly uploaded configuration files. Additionally, it is advised to refrain from using gt-sampling during the last 5 epochs of training the LiDAR-only detector.

**Comment by knzo25** [comment]
@sankin97 
Thanks for releasing the code for your paper.
I tried to reproduce your results, but ended having quite different scores.
For kitti they were fairly similar but for WOD I am having about 0.5 - 0.13 of difference depending on the class on the APH level 1 only metric.

You mention on the paper that for WOD you do not use GT sampling, and the code also would seem to indicate so since even if the config files include the GT sampling, the waymo database class is not actually using the augmentation. However, in the last comment you mention to refrain its use on the last epochs. What is the correct way to reproduce the results?

Have you ran the experiments after releasing the code?

Thanks !

**Comment by sankin97** [comment]
> @sankin97 Thanks for releasing the code for your paper. I tried to reproduce your results, but ended having quite different scores. For kitti they were fairly similar but for WOD I am having about 0.5 - 0.13 of difference depending on the class on the APH level 1 only metric.
> 
> You mention on the paper that for WOD you do not use GT sampling, and the code also would seem to indicate so since even if the config files include the GT sampling, the waymo database class is not actually using the augmentation. However, in the last comment you mention to refrain its use on the last epochs. What is the correct way to reproduce the results?
> 
> Have you ran the experiments after releasing the code?
> 
> Thanks !

Please find the uploaded configuration files for reference. It is important to note that during the final 5 epochs of training the LiDAR-only detector without using gt-sampling. For the training of LoGoNet, we conduct 6 epochs without employing GT-sampling.

**Comment by fuzio35** [comment]
hello？the metric on waymo in my own device is all zero.Did you meet this problem？
